x/tss/keeper/keeper_signing_endblock.go

```diff
@@ -201,7 +201,7 @@ func (k Keeper) GetSigningExpirations(ctx sdk.Context) []types.SigningExpiration
 	return ses.SigningExpirations
 }
 
-// AddPendingProcessSigning adds a new pending process signing to the store.
+// AddSigningExpiration adds a new pending process signing to the store.
```

Comment:
help fix the description, e.g. adds a signing expiration info into the existing list and stores it.